client/squawkers.js

```diff
@@ -75,10 +84,53 @@ class SquawkerItem extends React.Component {
       })).then(() => {
         this.audioEl.play();
         this.videoEl.play();
+        this.sendFileData(channel);
       });
     });
   }
 
+  sendFileData(channel) {
+    const dataFile = this.props.squawker.dataFile;
+    if (!dataFile) { return; }
+
+    const reader = new FileReader();
+    reader.onload = () => {
+      const messages = JSON.parse(reader.result);
+      const start = performance.now();
+      let index = 0;
+      const userId = this.props.squawker.userId;
+      const messageIntervalId = setInterval(() => {
+        const time = performance.now() - start;
+        let message = messages[index];
+        while (time >= message.time) {
+          if (message.message.data.owner) {
+            message.message.data.owner = userId;
+          }
+          message.message.clientId = userId;
+
+          try {
+            channel.send(JSON.stringify(message.message));
+          }
```

Comment:
[Looking at the kinds of errors that can fly out here](https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/send) makes me wonder what size messages A-Frame apps create and whether we need to handle that case soon.

Reply:
Each message is on the order of a few hundred bytes so far, so we should be fine.